python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/management-api-http.j2

```diff
@@ -12,7 +12,7 @@ management api http-commands
 {%         if management_api_http.https_ssl_profile is arista.avd.defined %}
    protocol https ssl profile {{ management_api_http.https_ssl_profile }}
 {%         endif %}
-{%     elif management_api_http.enable_https is arista.avd.defined(false) %}
+{%     else %}
    no protocol https
 {%     endif %}
 {%     if management_api_http.enable_http is arista.avd.defined(true) %}
```

Comment:
protocol http is not in correct order

Reply:
`no default-services` is not in correct order

Reply:
Fixed.